README.md

```diff
@@ -1,3 +1,6 @@
+# [DEPRICATED] 
+This repoisitory has been replaced with [app-services-sdk-core](https://github.com/redhat-developer/app-services-sdk-core) and should not be used. If you are using this SDK take a look at how to transition your project to the new SDK [here](https://github.com/redhat-developer/app-services-sdk-core/blob/main/MOVING.md).
```

Comment:
I think we should use a persistent link here.

Reply:
Done, we should also send out an email detailing the move and contact the teams we know that use it